sdk/src/main/java/io/dapr/client/domain/BulkAppResponse.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright 2023 The Dapr Authors
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package io.dapr.client.domain;
+
+import java.util.Collections;
+import java.util.List;
+
+/**
+ * Response from the application containing status for each entry in the bulk message.
+ * It is posted by the bulk subscribe handler.
+ */
+public class BulkAppResponse {
```

Comment:
Sorry for this but make sure this name is specific to PubSub. The current name is too generic.